app/helpers/application_helper/toolbar/network_routers_center.rb

```diff
@@ -15,41 +15,42 @@ class ApplicationHelper::Toolbar::NetworkRoutersCenter < ApplicationHelper::Tool
             t,
             :klass => ApplicationHelper::Button::NetworkRouterNew
           ),
-          separator,
-          button(
-            :network_router_edit,
-            'pficon pficon-edit fa-lg',
-            t = N_('Edit selected Router'),
-            t,
-            :url_parms => 'main_div',
-            :enabled   => false,
-            :onwhen    => '1'
-          ),
-          button(
-            :network_router_add_interface,
-            'pficon pficon-edit fa-lg',
-            t = N_('Add Interface to selected Router'),
-            t,
-            :url_parms => "main_div",
-            :enabled   => false,
-            :onwhen    => "1"),
-          button(
-            :network_router_remove_interface,
-            'pficon pficon-edit fa-lg',
-            t = N_('Remove Interface from selected Router'),
-            t,
-            :url_parms => "main_div",
-            :enabled   => false,
-            :onwhen    => "1"),
-          button(
-            :network_router_delete,
-            'pficon pficon-delete fa-lg',
-            t = N_('Delete selected Routers'),
-            t,
-            :url_parms => 'main_div',
-            :confirm   => N_('Warning: The selected Routers and ALL of their components will be removed!'),
-            :enabled   => false,
-            :onwhen    => '1+'),
+          # TODO: Uncomment when support? for list actions works on some degree
```

Comment:
why commit this?

Reply:
it's like that in other files. Once we deal with supports? these actions are working

Reply:
although in other files it references non related PR, from what the UI guys told me

Reply:
I'm not sure I follow, what do you mean by deal with support? bottom line - I can't see how adding in commented out code is acceptable?

Reply:
@Ladas: I know that you are referring to a situation where someone (bet it was not me) commented out some code.
As I see it, I'd rather delete the code.

On "Once we deal with supports? these actions are working" I don't think so or I don't see how. Please, explain.

I have explained a couple of times why we cannot have complex logic around enabling/disabling, showing/hiding toolbar buttons __in list views__ so I am not going to repeat it here again.

I think that the general approach should be:

 1. having only the __basic actions__ in the __list view__ toolbars and 
 1. having the __full range__ of functions in the toolbars on the __details screen__.

Reply:
Ah, so the actions are working now, since OpenStack was the only provider with Routers. I am adding AWS routers, so the actions will fail there.

Right, as I mentioned, it can be as as simple as assert_support line in each action, to move the check.

Ok, I will delete the commented code. My first try is always consistence. ;-)